domains/devendrakanojiya.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devendrakanojiya",
+        "email": "s_36100@bkbirlacollegekalyan.com"
+    },
+    "record": {
+        "TXT": "37b9724a41bf8e23b2a352d65b7d19"
```

Comment:
TXT records alone do nothing

Reply:
What more should I have to add ?

Reply:
was TXT record for github pages?

Reply:
yes

Reply:
you did it the wrong name, you needed to make a domain with the hostname github wanted and then put that txt

Reply:
should I have to add the hostname of the txt record ?

Reply:
can Message you on discord ?

Reply:
> you did it the wrong way, you needed to make a domain with the hostname github wanted and then put that txt


hostname is something like this :
_github-pages-challenge-devendrakanojiya.devendrakanojiya.is-a.dev  


So this basically means I have to make this long domain name 😭  instead of https://devendrakanojiya.is-a.dev

Reply:
> > you did it the wrong way, you needed to make a domain with the hostname github wanted and then put that txt
> 
> hostname is something like this : _github-pages-challenge-devendrakanojiya.devendrakanojiya.is-a.dev
> 
> So this basically means I have to make this long domain name 😭 instead of https://devendrakanojiya.is-a.dev

you'd need to make _github-pages-challenge-devendrakanojiya.devendrakanojiya.json for github pages verification (tho you need to own devendrakanojiya.is-a.dev first)

Reply:
oh great thanks! 👍 Mr.dog